tests/test_examples.py

```diff
@@ -1,20 +1,18 @@
-import logging
+from __future__ import print_function
 import tempfile
 
 import dill
+import theano
 
 import blocks
 from blocks.extensions.saveload import SAVED_TO
 from examples.sqrt import main as sqrt_test
 from examples.mnist import main as mnist_test
 from examples.markov_chain.main import main as markov_chain_test
+from examples.reverse_words import main as reverse_words_test
 from tests import silence_printing
 
-
-def setup():
-    # Silence Block's logger
-    logger = logging.getLogger(blocks.__name__)
-    logger.setLevel(logging.ERROR)
+theano.config.optimizer = 'fast_compile'
```

Comment:
We should probably put this in the `setUp` function and restore the default in `tearDown`, because right now this setting will persist for all tests, which might not be what we want, and in the future we might completely forget about it.